ControlRoom/Helpers/TypeIdentifier.swift

```diff
@@ -62,11 +62,11 @@ struct TypeIdentifier: Hashable {
             typesToCheck.append(contentsOf: first.conformsTo)
         }
 
-        return nil
+        return URL(fileURLWithPath: "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/GenericQuestionMarkIcon.icns")
```

Comment:
Instead of hard-coding the path, we can look it up dynamically:

```swift
let fileType = NSFileTypeForHFSTypeCode(OSType(kQuestionMarkIcon))!
let image = NSWorkspace.shared.icon(forFileType: fileType)
```

Reply:
Or alternatively:
```swift
let image = NSWorkspace.shared.icon(forFileType: "'ques'")
```